.github/scripts/templates/footer.md.jinja

```diff
@@ -0,0 +1,7 @@
+%% Hub footer: Please don't edit anything below this line %%
+
```

Comment:
```suggestion

## Footer
```

Reply:
@claremacrae What I meant was that by adding a 2nd level header here (and hiding just that line specifically, not its contents), we can use header embeds and that will exclude their contents on other pages. See here a manual example:  

![image](https://user-images.githubusercontent.com/5426039/147670896-2c6420cb-ea96-46ff-9e21-a0b30a3940ce.png)